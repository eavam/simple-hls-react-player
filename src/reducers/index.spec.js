import reducer from './index';

test('should return the initial state', () => {
  expect(reducer(undefined, {})).toMatchSnapshot();
});
