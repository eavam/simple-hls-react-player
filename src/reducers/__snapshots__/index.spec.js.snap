// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the initial state 1`] = `
Object {
  "currentTime": "00:00",
  "isFullscreen": false,
  "muted": false,
  "showControls": true,
  "typeActionPlayer": "play",
  "volume": 0,
  "volumeNumber": 0,
}
`;
